app/views/users/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<!--
+ <form action="/users" method="post" accept-charset="UTF-8">
```

Comment:
[required] It's better to write paths using rails way. Instead of `/users/:id` :x: it should be written  
 `users_path`:heavy_check_mark: